src/components/Card.jsx

```diff
@@ -1,26 +1,52 @@
+import { useState } from 'react';
+
 import { calculateDateDifference, formatDate } from '../utils/dateCalculate';
+
 import defaultImg from '../assets/logo.svg';
 import starIcon from '../assets/icon/star.svg';
 import kebabIcon from '../assets/icon/kebab.svg';
 
 import './Card.css'
+import ModalLinkAdd from './modals/ModalLinkAdd';
+
+const Card = ({imgSrc, createdAt, title, description, url, folderName, folderList}) => {
+  const [isOpenKebab, setIsOpenKebab] = useState(false);
+  const [isOpenModalLinkAdd, setIsOpenModalLinkAdd] = useState(false);
+
+  const handleClickKebab = () => {
```

Comment:
현재 함수명은 `UI에서 어떤 행위가 발생하는가?`에 대해서 표현되어 있는데요!
이 함수를 실행했을 때 어떤 동작이 일어나게 될지에 대해서는 표현이 되어있지 않아서 함수명을 보고 동작을 유추하기에는 어렵다는 아쉬움이 있을 것 같아요~ 
따라서 이렇게 표현해줘도 좋을 것 같습니다~
```suggestion
  const handleClickOpenKebab = () => {
```